nocasedict/_nocasedict.py

```diff
@@ -375,9 +399,13 @@ def update(self, *args, **kwargs):
 
         The updates are performed first for the positional arguments in the
         iteration order of their iterables, and then for the keyword arguments.
-        Note that before Python 3.4, keyword arguments are passed to this
-        method as a standard dict, so the order of updates for the keyword
-        arguments is not preserved.
+
+        Starting with Python 3.7, the order of keyword arguments as specified
```

Comment:
Again question. Doesn't this dict do that for all python versions?

Reply:
See above.